### DISCUSSION TITLE
Header Background Color

### DISCUSSION THREAD

**Discussion by aviau** [dolanmiu/docx#1608] [General]
Hello!

Is it possible to set the background color of a header?

**Comment by ElliottJPerry** [comment]
Did you figure out a workaround @aviau?

**Comment by ElliottJPerry** [comment]
For anyone seeing this, I don't believe it is possible to set the background colour of a header that extends to each edge (as if a custom letterhead design, for example). As a workaround, I created an image and positioned the image appropriately with no gap on the left, top or right edge.
